security-response.md

```diff
@@ -0,0 +1,102 @@
+branch at the moment of the release. For instance, if the latest version is
+0.10.0, bug fixes are released either as part of 0.11.0 or 0.10.1.
+
+Security fixes are given priority and might be enough to cause a new version to
+be released. Each repository is entitled to establish their own complementary
+processes. SIG-Security in conjunction with the TC can advise in case
+clarifications are required.  
+
+## Disclosures
+
+### Private Disclosure Processes
+
+In order for the vulnerability reports to reach maintainers as soon as possible,
+the preferred way is to use the `Report a vulnerability` button on the
+`Security` tab in the respective GitHub repository. It creates a private
+communication channel between the reporter and the maintainers.
+
+If you are absolutely unable to or have strong reasons not to use GitHub
+reporting workflow, please reach out to the Technical Committee using
+[cncf-opentelemetry-tc@lists.cncf.io](mailto:cncf-opentelemetry-tc@lists.cncf.io)
+and we will provide instruction on how to report the vulnerability using an
+encrypted message, if desired.
+
+[gh-organization]: https://github.com/open-telemetry
+
+### Public Disclosure Processes
+
+If you know of a publicly disclosed security vulnerability please IMMEDIATELY
+email
+[cncf-opentelemetry-tc@lists.cncf.io](mailto:cncf-opentelemetry-tc@lists.cncf.io)
+to inform the Security Response Committee (SRC) about the vulnerability so they
+may start the patch, release, and communication process.
+
+The TC should receive the message and re-direct it to the relevant repo
+maintainers for ownership. If possible the repo maintainers will engage and ask
+the person making the public report if the issue can be handled via a private
+disclosure process. If the reporter denies the request, the repo maintainers
+will move swiftly with the fix and release process. In extreme cases you can ask
+GitHub to delete the issue but this generally isn't necessary and is unlikely to
+make a public disclosure less damaging.
+
+## Patch, Release, and Public Communication
+
+For each vulnerability, a member of the TC will volunteer track issue resolution
+with the relevant repo owners. This TC member will be referred to as the Fix
+Lead and is not responsible for incident remediation or implementation.
+
+The role of Fix Lead should rotate round-robin across the TC.
+
+### Fix Team Organization
+
+The Fix Team is made up of the Fix Team Lead and the relevant repo maintainers as Fix Responders.
+
+### Fix Development Process
+
+All of the timelines below are suggestions that assume a Private Disclosure and that the report is accepted as valid. These steps should be completed within the 1-7 days of Disclosure.
+
+- The TC is notified of an incident, a member volunteers to track the issue as Fix Lead, and the relevant repo maintainers are added as the Fix Team to the issue.
+- The Fix Responders will create a
+  [CVSS](https://www.first.org/cvss/specification-document) score using the
```

Comment:
We should clarify if we're just looking for the base score or if there's some reason we want the temporal and/or environmental scores as well

Reply:
limiting to base score for now